docs/SMCWG-charter.md

```diff
@@ -287,18 +291,22 @@ three (3).
   to the Chair herein MAY be performed by the Vice Chair. For example, the Vice
   Chair MAY preside at SMCWG Meetings and Teleconferences in the Chair’s
   absence.
-- SMCWG-created Subcommittees MAY be approved either:
-
-  1. by formal ballot as described in [6.1](#6.1-voting-structure), or
-  2. by simple majority vote of those members present at a regularly
-     scheduled SMCWG Meeting or Teleconference provided that the proposal is
-     mentioned in an agenda circulated on the SMCWG Mail List at least
-     forty-eight (48) hours prior to the SMCWG Meeting or Teleconference.
+- The SMCWG may establish subcommittees, by formal ballot, to address any of the
+  SMCWG’s business as specified in the ballot (each, a “SMCWG Subcommittee”).
+  SMCWG Subcommittees are open to all SMCWG Members. A SMCWG Subcommittee may
+  work on and recommend ballots, complete delegated functions, or issue reports
+  to the SMCWG that are within the scope of this charter. SMCWG Subcommittees
+  must post all agendas and minutes on a public mail list.
 
 ### 7. Summary of Major Deliverables
 
-The deliverables of the SMCWG are defined in the Scope section above and shall
-be named the "Baseline Requirements for S/MIME Certificates" proposed standard.
+The primary deliverable of the SMCWG is set forth in the Scope section, and
+shall include a Draft Guideline be named the "Baseline Requirements for S/MIME
+Certificates".
+
+Following the adoption of such a Draft Guideline, the SMCWG MAY produce
```

Comment:
Again, as I've stated at various F2Fs, we can't support sequencing.

Reply:
@timfromdigicert It's useful if you could actually point to that somewhere in the minutes during the past discussions, so we can see if there's a productive path forward.

To use the language from https://github.com/cabforum/documents/pull/167#discussion_r394636671 , here's the discussion:

https://cabforum.org/2018/10/18/minutes-for-ca-browser-forum-f2f-meeting-45-shanghai-17-18-october-2018/
> **Ben** – It sounds like the initial charter should focus on three aspects: profile, identity validation of email and identity (host and local part), and private key protection.
> Kirk Hall, Entrust – Is that enough to start drafting a charter?
> **Ben** – **Yes, I can start a charter based on those three principles.**

This is Ben committing to (1)

https://cabforum.org/2019/08/16/minutes-for-ca-browser-forum-f2f-meeting-47-thessaloniki-12-13-june-2019/
> **Dean Coclin** (recapping differences): In Cupertino, talked about the charter. One of the issues was the scope of the charter, and the inclusion of validation of identities and not just domains. Ryan had suggested limiting the charter to just domains. **Dean’s compromise solution was to define in the charter that domain validation should be complete before taking on identity, but adding identity.**
> **Ryan Sleevi**: Two areas of debate about charter scope. One challenge in the scope was whether we include the possibility of discussing other forms of identity, such as natural or corporate identity, or exclude it from the charter. Another challenge in the scoping was whether or not it’s limited to S/MIME, or given the similarities between identity and document signing and other cases, whether those other cases were included, which had been raised by some of the European members where they may use similar processes, especially with respect to validating client identity. Suggestion from Cupertino was to exclude identity wholesale, once we’ve got the S/MIME portion of domain/e-mail validation nailed down. The other identity cases are different consumers.
> **Tim Hollebeek**: Yeah, it’s likely we’ll eventually having WGs to tackle some of those, because they’re important. **I don’t think anyone has proposed they should be in scope.**
> ...
> **Tim Hollebeek**: So **is Dean’s proposed path viable?**

This is Dean proposing (2), and you seemingly on board

https://cabforum.org/2019/12/12/minutes-for-ca-browser-forum-f2f-meeting-48-guangzhou-5-7-november-2019/
> Dean – What will Google do if we cover email address validation first for Phase 1, and then Phase 2 followed up by identity? Does Google participate for Phase 1 and then “I’m out” during Phase 2 or is there IP review
> Ryan explained the ideal outcome was that by the time the WG was ready to tackle identity, a better sense of the obligations could be done. Focusing on e-mail validation helps narrowly scope the IP commitments, making it easier to review and engage, but that the broad scope of identity means substantially more work to participate.

Here's me explaining that (1) is preferable to (2), because it allows immediate participation, but that (2) is still viable.

If I'm understanding, you're now insisting (3) is the only option DigiCert will support. I'm not sure what changed, especially when two DigiCert representatives were previously supportive of (1) and (2), respectively, which is part of why the (as written) (3) was and is such a surprise and concern.

This is trying to capture (2)

Reply:
@timfromdigitcert ping on this

Reply:
Hey @timfromdigicert, I wanted to try and reach out again to see if there's a possible path forward here, in advance of Thursday's call.

It would be great to make some progress on this, as it addresses the feedback discussed in https://github.com/cabforum/documents/pull/167 and the added feedback from @clintwilson and @wthayer